backend/domain/domainmodels.js

```diff
@@ -0,0 +1,119 @@
+class Room {
+    constructor() {
+
+        this.roomName = this.setRoomName(); // some function to initialise a roomID
+
```

Comment:
Also now since its class-based. You don't have to do `this.groupName = function () { ....}`. You can do it like a normal method like java so  ` groupName() {... }`

Reply:
changed along with moving all methods outside